.github/workflows/config/reviewdog.yaml

```diff
@@ -12,3 +12,12 @@ runner:
     errorformat:
       - '%A%f:%l: %m'
       - '%C%.%#'
+  license-yaml:
+    cmd: >-
+      boilerplate-check check
+      --boilerplate=hack/boilerplate/boilerplate.yaml.txt
+      --file-extension=yaml
+      --exclude='(^(\.|config/samples/))'
```

Comment:
I wasn't sure whether we wanted to enforce headers inside samples or not.
Thoughts?

Reply:
+1 they should be there

Reply:
No strong opinion but mostly concur with Jeff.

Reply:
Thanks for the comments!
I added missing headers to all samples reported by the linter.